data/json/items/melee/spears_and_polearms.json

```diff
@@ -432,6 +432,46 @@
     "price_postapoc": 500,
     "qualities": [ [ "COOK", 1 ] ]
   },
+  {
+    "id": "pike_wood",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": { "ctxt": "weapon", "str": "wooden pike" },
+    "description": "This is a medieval weapon consisting of a wood shaft with a fire hardened point.",
+    "weight": "2300 g",
```

Comment:
This pike loses more than a kilogram after processing from a long stick (3.4 kg)?